wolfcrypt/src/tfm.c

```diff
@@ -2430,7 +2430,7 @@ static int _fp_exptmod_nct(fp_int * G, fp_int * X, fp_int * P, fp_int * Y)
   fp_int  *res;
   fp_digit buf, mp;
   int      err, bitbuf, bitcpy, bitcnt, mode, digidx, x, y, winsize;
-#ifndef WOLFSSL_NO_MALLOC
+#ifdef WOLFSSL_SMALL_STACK
```

Comment:
This change makes sense. Can you please use `#if defined(WOLFSSL_SMALL_STACK) && !defined(WOLFSSL_NO_MALLOC)`. It ensures both cases are covered, even though small stack and no malloc are not compatible.